.evergreen.yml

```diff
@@ -4,20 +4,104 @@ ignore:
   - "*.md"
   - ".github/*"
 
+modules:
+- name: evergreen
+  repo: git@github.com:evergreen-ci/evergreen.git
+  branch: master
+  prefix: gopath/src/github.com/evergreen-ci
+
+#######################################
+#              Functions              #
+#######################################
 functions:
   get-project:
     command: git.get_project
     type: setup
     params:
       directory: spruce
 
-  npm-install:
+  get-evergreen-project:
+    command: git.get_project
+    type: setup
+    params:
+      directory: spruce
+      token: ${github_token}
```

Comment:
This token won't be populated without the setup code in evergreen, but you don't actually need it

Reply:
it's an environment variable

Reply:
I had missed it being set as a project expansion, but it's still not needed unless you specifically want to clone over HTTPS (normally necessary if this is running in a docker container)